### PR TITLE
Merch components dismiss 404 test

### PR DESCRIPTION
**Pull request by uplne**
# It's there, it's not, it's there...
## What does this change?

Nothing. However it will tell us how much users want this feature.
## Does this affect other platforms - Amp, Apps, etc?

Possibly. In the future.
## Screenshots

screen shot 2016-03-04 at 15 46 27
screen shot 2016-03-04 at 15 52 24
screen shot 2016-03-04 at 15 59 07
screen shot 2016-03-04 at 15 52 52

### CODE REVIEW

File: commercial/app/views/static/surveySimplePageMembership.scala.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+        <div class="l-side-margins">
+
+            <div class="live-events-survey-wrapper" data-link-name="live events survey page">
+
+                <div class="live-events-survey-wrapper-inner">
+
+                    @fragments.inlineSvg("survey-thumbsup", "commercial")
+
+                    <h1>You're ahead of your time.</h1>
```
Comment (regiskuckaertz) [inline commercial/app/views/static/surveySimplePageMembership.scala.html]: <typography snob>
You’re, not You're
</typography snob>

:sob:

File: static/src/javascripts/projects/common/modules/commercial/survey/survey-simple.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+                crossIcon: svgs('crossIcon'),
+                surveyOverlaySimple: svgs('surveyOverlaySimple')
+            });
+    };
+
+    surveySimple.prototype.attach = function () {
+        fastdom.write(function () {
+            $(document.body).append(this.bannerTmpl);
+
+            bean.on(document, 'click', $('.js-survey-close'), function () {
```
Comment (regiskuckaertz) [inline static/src/javascripts/projects/common/modules/commercial/survey/survey-simple.js]: I've seen this pattern many times. Any advantage compared to just binding the listener to the close button itself?

File: commercial/app/views/static/surveySimplePageMembership.scala.html (hunk 1 above)
Comment (jamesgorrie) [inline commercial/app/views/static/surveySimplePageMembership.scala.html]: Grammatically, for contractions, we should be using an apostrophe - so it more about having the font render apostrophes correctly?

File: static/src/javascripts/projects/common/modules/commercial/survey/survey-simple.js (hunk 2 above)
Comment (jamesgorrie) [inline static/src/javascripts/projects/common/modules/commercial/survey/survey-simple.js]: Deferring events to parents is a performance thing as far as I remember (I can't find any citations just yet). The theory being you would only have one event listener that has a `case` in it.

Whether this is still a performance consideration - I am not sure? @sndrs?

File: static/src/stylesheets/module/commercial/surveys/_survey.scss [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+        background-color: #01558b;
+        color: #ffffff;
+        box-sizing: border-box;
+
+        @include mq(tablet) {
+            left: 50%;
+            top: percentage(1/5);
+            width: $gs-column-width * 8;
+            height: auto;
+            margin-left: -180px;
```
Comment (regiskuckaertz) [inline static/src/stylesheets/module/commercial/surveys/_survey.scss]: I suppose you could write it `$gs-column-width * 4` as well

File: static/src/javascripts/projects/common/modules/commercial/survey/survey-simple.js (hunk 2 above)
Comment (uplne) [inline static/src/javascripts/projects/common/modules/commercial/survey/survey-simple.js]: Event delegation

File: commercial/app/views/static/surveySimplePageMembership.scala.html [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+                <div class="footer-area">
+
+                    <div class="live-events-survey-wrapper-inner">
+
+                        <ul class="u-unstyled l-row l-row--cols-2">
+
+                            <li class="l-row__item l-row__item--span-1">
+
+                                <p>If you would like to become a Member to support the Guardian, the button below works. Really.</p>
```
Comment (Calanthe) [inline commercial/app/views/static/surveySimplePageMembership.scala.html]: Are we really going to show that text?

File: static/src/stylesheets/module/_adslot.scss [hunk 5]
```diff
@@ -656,3 +656,28 @@
         }
     }
 }
+
+.ad-slot--dimiss {
```
Comment (regiskuckaertz) [inline static/src/stylesheets/module/_adslot.scss]: `.ad-slot__dismiss`, no?

File: commercial/app/views/static/surveySimplePageSignup.scala.html [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+
+            <div class="live-events-survey-wrapper" data-link-name="live events survey page">
+
+                <div class="live-events-survey-wrapper-inner">
+
+                    @fragments.inlineSvg("survey-thumbsup", "commercial")
+
+                    <h1>You're ahead of your time.</h1>
+
+                    <p class="standfirst">Sorry, but this feature isn't available yet. This is part of a test to gauge interest in new and upcoming features. So thank you, you have just helped improve the user experience of our site.</p>
```
Comment (jamesgorrie) [inline commercial/app/views/static/surveySimplePageSignup.scala.html]: If this feature becomes available - do we let them know about it?

File: commercial/app/views/static/surveySimplePageMembership.scala.html (hunk 4 above)
Comment (uplne) [inline commercial/app/views/static/surveySimplePageMembership.scala.html]: Yes, that is the approved message.

File: commercial/app/views/static/surveySimplePageSignup.scala.html (hunk 6 above)
Comment (uplne) [inline commercial/app/views/static/surveySimplePageSignup.scala.html]: The decision from product owner was not to get user emails this time. It wasn't that significant number of users last time we did this.

**Comment by regiskuckaertz** [comment]
:+1:

**Pull request merged**